pkg/util/certificate/certificate.go

```diff
@@ -0,0 +1,30 @@
+package certificate
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"crypto/x509"
+	"math"
+	"time"
+)
+
+const DefaultMinDurationPercent = 0.20
+
+// lessThanMinimumDuration indicates whether the provided cert has less
+// than the provided minimum percentage of its duration remaining.
+func LessThanMinimumDuration(cert *x509.Certificate, minDurationPercent float64) bool {
+	expiry := cert.NotAfter
```

Comment:
These function names should be consistent since they're both returning `bool`. Either rename this one `IsLessThanMinimumDuration` (or something similar but shorter like `IsStale`) or rename the other one `CertExpired`.

Reply:
Done.